app/src/main/res/values/styles.xml

```diff
@@ -44,4 +44,29 @@
         <item name="android:background">@drawable/qr_code</item>
     </style>
 
+    <!-- Home activity -->
+    <style name="HomeActivity">
+
+    </style>
+
+    <style name="HomeActivity.Margins">
```

Comment:
Ovako kad gledam iz imena, kao čitatelju mi nije u potpunosti jasna ideja ovog style-a. Može neki info?

Reply:
Nekako sam se ravnao prema CSS-u i klasama koje tamo ima. 
S obzirom da više elemenata u xml-u ima iste margine, to sam stavio u neki "razred" koji onda može naslijediti specifični stil, 
npr: imam UserNameTextView i EmailTextView koji imaju iste marginei nalaze se u istom activityju. U tom slučaju će oba stila biti nazvana HomeActivity.Margins.SPECIFIC gdje je SPECIFIC ime tog elementa.